v22.2/drop-owned-by.md

```diff
@@ -0,0 +1,38 @@
+---
+title: DROP OWNED BY
+summary: The DROP OWNED BY statement drops all objects owned by and any grants on objects not owned by a role.
+toc: true
+docs_area: reference.sql
+---
+
+The `DROP OWNED BY` [statement](sql-statements.html) drops all objects owned by and any grants on objects not owned by a role.
+
+{% include {{ page.version.version }}/misc/schema-change-stmt-note.md %}
+
+## Required privileges
+
+The role must have the `DROP` [privilege](security-reference/authorization.html#managing-privileges) on the specified objects. `DROP OWNED BY` cannot be performed if the
+user has synthetic privileges.
```

Comment:
From what I'm seeing from the code base, a `Synthetic Privileges` is represented by a in memory privilege descriptor, it contains privilege information for anything that is not descriptor backed (virtual table, global grant etc...): https://github.com/cockroachdb/cockroach/blob/87ed064dc23eab6948ee8a07e8507f150bda0e44/pkg/sql/privilege/privilege.go#L108

Reply:
I need something that defines this in terms of language that is not implementation dependent.